tests/test_imphooks.py

```diff
@@ -10,7 +10,9 @@
 from tools import mock_xonsh_env
 LOADED_HERE = False
 
-def setup_module():
+IMP_ENV = {'PATH': [], 'PATHEXT': []}
+
+def setup():
```

Comment:
it was a conflict with the master merge and i kept the master part, will change it